packages/nuejs.org/docs/index.md

```diff
@@ -19,13 +19,13 @@ Nue's content-first [development flow](ux-development.html) focuses solely on th
 - - -
 
 ## Target audience
-Nue is a great fit for the following people:
+Nue is a great fit for the following group of people:
 
 1. **UX developers**: who prefer to jump to CSS directly from **Figma** or **Sketch** without a [frustrating and time-consuming](//medium.com/design-warp/5-most-common-designer-developer-handoff-mishaps-ba96012be8a7) designer-developer handoff process in the way.
 
-2. **Beginner web developers**: who want to skip the redundant layers in the [frontend stack](//roadmap.sh/frontend) and start building websites quickly with modern HTML, CSS, and JavaScript.
+2. **Beginner web developers**: who want to skip the redundant layers in the [frontend stack](//roadmap.sh/frontend) and start building websites quickly with modern HTML, CSS and JavaScript.
```

Comment:
I'd like to merge this, but the oxford comma bothers me. Should be either everywhere, or not. 

https://www.scribbr.com/commas/oxford-comma/

Reply:
The Oxford comma is optional—it’s not a grammatical error to leave it out or to include it.
If you feel that it should be there, please include it.